spec/lib/govuk_publishing_components/app_helpers/countdown_helper_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "spec_helper"
+
+RSpec.describe GovukPublishingComponents::AppHelpers::CountdownHelper do
+  describe "Countdown helper" do
+    it "gives the days left until end of transition period" do
+      clock = GovukPublishingComponents::AppHelpers::CountdownHelper.new
```

Comment:
you can make this a bit less verbose with `described_class.new`

Reply:
Nice one! Updated